xbmc/cores/RetroPlayer/playback/ReversiblePlayback.cpp

```diff
@@ -193,6 +207,7 @@ bool CReversiblePlayback::LoadSavestate(const std::string& path)
     if (m_gameClient->Deserialize(savestate->GetMemoryData(), memorySize))
     {
       m_totalFrameCount = savestate->TimestampFrames();
+      m_loadedSavestatePath = path;
```

Comment:
In the future, we might want to know more about the loaded savestate beyond the path. It will be more scalable if we make the savestate a `std::shared_ptr` and store the entire thing, and replace `m_loadedSavestatePath` with `m_loadedSavestate->Path()`.

Reply:
Ah I see, the savestate doesn't save its own path, this has to be stored outside the savestate. Disregard comment.